test/extensions/filters/http/wasm/test_data/body_cpp.cc

```diff
@@ -34,27 +34,61 @@ FilterHeadersStatus ExampleContext::onResponseHeaders(uint32_t) {
   return FilterHeadersStatus::Continue;
 }
 
-void ExampleContext::logBody(BufferType bt) {
```

Comment:
This isn't go, s/bt/buffer_type/

Reply:
That's and old code that I'm fixing here.